config/interceptors/core-interceptors.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: Interceptor
+metadata:
+  name: cel
+spec:
+  clientConfig:
+    url: "http://tekton-triggers-core-interceptors.tekton-pipelines.svc/cel"
+  # Until #869 is implemented, the params field is only informational
+  params:
+    - name: "filter"
+      description: "A CEL expression that evaluates to true or false"
+    - name: "overlays"
+      description: "CEL expressions whose values get mapped to user defined keys"
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: Interceptor
+metadata:
+  name: bitbucket
+spec:
+  clientConfig:
+    url: "http://tekton-triggers-core-interceptors.tekton-pipelines.svc/bitbucket"
```

Comment:
just thinking at high level

These interceptors are core interceptors which will be installed as part of triggers installation what if Tekton installs in different namespace other than `tekton-pipelines` :thinking: 

In that case will this be responsibility of user/admin who installs Triggers ?

Reply:
yeah this is a bit confusing right now. The idea is instead of url we'd also support a `service` field that contains a reference to the service for the deployed core interceptors. Custom installs would have to change the namespace of the service but would not have to modify a URL.

(I was thinking of adding that as a separate PR but maybe I'll add it to this one itself to make it less confusing).